dspace/config/modules/ldn.cfg

```diff
@@ -15,16 +16,16 @@ ldn.notify.local-inbox-endpoint = ${dspace.server.url}/ldn/inbox
 # Each IDs must match with the ID returned by the external service 
 # in the JSON-LD Actor field
 
-service.service-id.ldn = dev-hdc3b.lib.harvard.edu/api/inbox
+service.service-id.ldn = hdc-dash-api-dev.lib.harvard.edu/api/inbox
```

Comment:
Presumably `dev`, `QA`, and `Prod` will all have different values for these variables... managed by our Ansible config. Will that work here?

Reply:
Yes, and that is defined by the IF project local.cfg files. I wanted the defaults to be dev for example.